src/insert-extension.js

```diff
@@ -59,8 +59,10 @@ function createAmpExtensionScript(win, extension) {
   ampExtensionScript.setAttribute('custom-element', extension);
   ampExtensionScript.setAttribute('data-script', extension);
   const pathStr = win.location.pathname;
+  const useCompiledJs = window.ampTestRuntimeConfig ?
```

Comment:
This is certainly fine. But for "shortness" you can also do this:
```
const useCompiledJs = window.ampTestRuntimeConfig &&
    window.ampTestRuntimeConfig.useCompiledJs;
```

This is a bit more idiomatic...